wireguard issue

again error After update

```
Building the wireguard kernel module...
getconf: LFS_CFLAGS: unknown variable
getconf: LFS_LDFLAGS: unknown variable
getconf: LFS_LIBS: unknown variable
  CC [M]  /wireguard/src/main.o
/bin/sh: scripts/basic/fixdep: not found
make[2]: *** [scripts/Makefile.build:334: /wireguard/src/main.o] Error 127
make[1]: *** [Makefile:1552: _module_/wireguard/src] Error 2
make: *** [Makefile:36: module] Error 2
```

--- a/tor-browser/alpha/Dockerfile
+++ b/tor-browser/alpha/Dockerfile
@@ -33,7 +33,7 @@ RUN useradd --create-home --home-dir $HOME user \
 ENV LANG C.UTF-8
 
 # https://www.torproject.org/projects/torbrowser.html.en
-ENV TOR_VERSION 8.5a3
+ENV TOR_VERSION 8.5a6
 ENV TOR_FINGERPRINT 0x4E2C6E8793298290
 
 # download tor and check signature

